File: Python/LAB 1/Course.cpp
#include <string>
#include <iostream>
#include "Course.h"

using namespace std;

int Course::numberOfCourses = 0;

Course::Course(string name, string semester)
: name(name), semester(semester)
{
    numberOfCourses++;
    code = numberOfCourses;
}

string Course::getName() {
    return name;
}

int Course::getCode() {
    return code;
}

string Course::getSemester() {
    return semester;
}

string Course::getInfo() {
    string info = name + " " + semester + to_string(code) ;
    return info;
}

File: LAB 1/Course.py
class Course:
    
    numberOfCourses = 0
    
    def __init__(self, name, semester):
        self.name = name
        self.semester = semester
        
        self.studentsInCourse = []
        
        Course.numberOfCourses += 1
        self.code = Course.numberOfCourses
        
    def __str__(self):
        return f"{self.name}, {self.semester}, {self.code}"
        
    def getName(self):
        return self.name

    def getSemester(self):
        return self.semester
    
    def getCode(self):
        return self.code
    
    def getInfo(self):
        print(f"Name: {self.name}\nSemester: {self.semester}\nCode: {self.code}")
        
    def addStudent(self, student):
        self.studentsInCourse.append(student)
        